File: abc/abc125/abc125_b.cs
// https://atcoder.jp/contests/abc125/submissions/8422748

using System;
using System.Linq;

namespace CSharp
{
    class Program
    {
        public static void Main()
        {
            var N = int.Parse(Console.ReadLine());

            var V = Console.ReadLine().Split(' ').Select(int.Parse).ToArray();
            var C = Console.ReadLine().Split(' ').Select(int.Parse).ToArray();

            Console.WriteLine(V.Zip(C, (v, c) => (v - c)).Sum(x => x > 0 ? x : 0));
        }
    }
}

File: abc/abc125/abc125_b.fs
// https://atcoder.jp/contests/abc125/submissions/12276077

let _ = stdin.ReadLine()
let V = stdin.ReadLine().Split() |> Array.map int
let C = stdin.ReadLine().Split() |> Array.map int

Array.zip V C
|> Array.map (fun x -> fst x - snd x)
|> Array.filter (fun x -> x > 0)
|> Array.sum
|> stdout.WriteLine
